README.md

```diff
@@ -13,24 +13,30 @@ play with it and give us your feedback :)
 
 ## Installation
 
-You need to have Java (>= 11) and Node (>= 16) and npm installed.
+The recommended way to install Effekt is by running:
+```bash
+npm install -g @effekt-lang/effekt
+```
+This will download the compiler and install the `effekt` command to the directory `npm root -g`.
 
-The recommended way of installing Effekt is by running:
+You need to have Java (>= 17) and Node (>= 16) and npm installed.
 
-```
-npm install -g https://github.com/effekt-lang/effekt/releases/latest/download/effekt.tgz
+#### Install specific release
+
+Alternatively, you can also download a specific release on the
+[release page on Github]({{ site.githuburl }}/releases).
+And then install it with
+```bash
+npm install -g <PATH_TO_FILE>/effekt.tgz
 ```
```

Comment:
Do we still need this? Can you use `npm` to install a specific version?

Reply:
I'd still keep this, but we might want to put it into a `<details>` section.